node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
Sorry this fell off my radar.

While it is true that the name setResourcePath is not the best terminology, we can't rename the function in a patch-update and I think two functions would be more confusing. We can add intellisense comments to the function.

It's atypical for a task to import multiple resource files. The problem we are actually running into is that a task imports a resource file, and then a module that it references imports a separate resource file. From the task author perspective, they are only importing one resource file. From the module author perspective, they also are only importing one resource file. It's important in the function intellisense comments to indicate it overlays and that authors of re-usable modules should prefix their resource keys to avoid collisions (for instance all task lib resource keys are prefixed `LIB_`)